frontend/src/app/game/simulation-control/simulation-control.component.ts

```diff
@@ -32,16 +32,21 @@ export class SimulationControlComponent implements OnInit {
       untilDestroyed(this),
     ).subscribe(speed => {
       this.speed = speed;
-      this.cd.markForCheck();
+      this.cd.detectChanges();
```

Comment:
Why this, pls?

Reply:
Sry, pls, unify, make better, no good reason for this.